helm-chart/renku/templates/tests/test-renku.yaml

```diff
@@ -80,11 +80,11 @@ spec:
     {{ end }}
     resources:
       requests:
-        memory: "4G"
-        cpu: "2"
+        memory: "3G"
+        cpu: "1500m"
         ephemeral-storage: "1G"
       limits:
-        memory: "4G"
-        cpu: "2"
+        memory: "3G"
+        cpu: "1000m"
```

Comment:
Is it normal that the limit is lower than the request?

Reply:
is normally the other way around, probably was a typo...

Reply:
oh gosh, I seem to be unable to edit these properly through the gitlab web UI...